airspace/airspace_list.txt

```diff
@@ -27,3 +23,5 @@ za openair http://soaringweb.org/Airspace/ZA/RSA_Airspace_17AUG2017.txt
 au openair http://soaringweb.org/Airspace/AU/australia_class_all_no_ctaf_e_class_freq.bnd_area_freq.bnd_17_11_09.txt
 ca openair http://soaringweb.org/Airspace/NA/canadian_airspace_noE_below12500_265.air
 fi openair http://www.hansto.fi/seeyou/files/Ilmatila2012.txt
+cz openair https://github.com/XCSoar/xcsoar-data-content/blob/master/airspaces/Czech_Airspace.txt
```

Comment:
this is a link to a webpage. we need a link to the raw file.

do we have permission to use and redistribute the data?

Reply:
- The link to the raw file is in the pull request, it's:
https://github.com/XCSoar/xcsoar-data-content/blob/master/airspaces/Czech_Airspace.txt

- I've uploaded the files to the XCSoar repository myself and have no idea about permissions, what would you need?

Reply:
that link is not a raw file. it links to a webpage that is showing the file.

> what would you need?

permission from the creator that we are allowed to use and redistribute the data.